### PR TITLE
Skeleton impl for tournaments

### PR DESCRIPTION
**Pull request by john-mikhail**
Some stuff added that is not used yet, but will be.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
     "paper-menu-button": "PolymerElements/paper-menu-button#~1.0.3",
     "paper-tabs": "PolymerElements/paper-tabs#~1.0.4",
     "paper-tooltip": "PolymerElements/paper-tooltip#^1.1.2",
-    "polymer": "Polymer/polymer#^1.6.1"
+    "polymer": "Polymer/polymer#^1.6.1",
+    "jquery-bracket": "^0.10.1"
```
Comment (pylaligand) [inline bower.json]: move up to keep the aB order

File: public/elements/zero-tournaments.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+      ul {
+        list-style: none;
+        margin: 0;
+        padding: 0;
+      }
+    </style>
+
+    <p>
+      Want to match up against your fellow clan menbers??
```
Comment (pylaligand) [inline public/elements/zero-tournaments.html]: members

File: public/index.html [hunk 3]
```diff
@@ -154,6 +155,19 @@
     <app-box class="first" effects="parallax-background"></app-box>
   </section>
 
+  <!-- Tournament -->
+  <section>
```
Comment (pylaligand) [inline public/index.html]: I wouldn't worry about adding it to the front page just yet. We could just have the single tournament at https://www.hundredzeros.net/zeromania.

File: bower.json (hunk 1 above)
Comment (john-mikhail) [inline bower.json]: done

File: public/elements/zero-tournaments.html (hunk 2 above)
Comment (john-mikhail) [inline public/elements/zero-tournaments.html]: Done.

**Pull request merged**